config/configmaps/install-log-regexes-configmap.yaml

```diff
@@ -42,6 +42,11 @@ data:
       - "Throttling: Rate exceeded"
       installFailingReason: AWSAPIRateLimitExceeded
       installFailingMessage: AWS API rate limit exceeded
+    - name: GeneralQuota
+      searchRegexStrings:
+      - "Quota '[A-Z_]*' exceeded"
+      installFailingReason: AWSQuotaExceeded
+      installFailingMessage: AWS Quota exceeded
```

Comment:
@twiest Is this for AWS only or can happen for any cloud? I got one today for GCP with similar error message.
```
level=info msg="Credentials loaded from environment variable \"GOOGLE_CREDENTIALS\", file \"/.gcp/osServiceAccount.json\""
    level=info msg="Consuming Install Config from target directory"
    level=info msg="Manifests created in: manifests and openshift"
    level=warning msg="Found override for release image. Please be warned, this is not advised"
    level=info msg="Consuming OpenShift Install (Manifests) from target directory"
    level=info msg="Consuming Common Manifests from target directory"
    level=info msg="Consuming Worker Machines from target directory"
    level=info msg="Consuming Master Machines from target directory"
    level=info msg="Consuming Openshift Manifests from target directory"
    level=info msg="Ignition-Configs created in: . and auth"
    level=info msg="Credentials loaded from environment variable \"GOOGLE_CREDENTIALS\", file \"/.gcp/osServiceAccount.json\""
    level=info msg="Consuming Master Ignition Config from target directory"
    level=info msg="Consuming Bootstrap Ignition Config from target directory"
    level=info msg="Consuming Worker Ignition Config from target directory"
    level=warning msg="Failed to find information on quotas compute.googleapis.com/cpus, compute.googleapis.com/regional_static_addresses"
    level=info msg="Creating infrastructure resources..."
    level=error
    level=error msg="Error: Error waiting for instance to create: Quota 'SSD_TOTAL_GB' exceeded.  Limit: 500.0 in region asia-northeast2."
    level=error
    level=error
    level=error msg="  on ../tmp/openshift-install-944165972/bootstrap/main.tf line 44, in resource \"google_compute_instance\" \"bootstrap\":"
    level=error msg="  44: resource \"google_compute_instance\" \"bootstrap\" {"
    level=error
    level=error
    level=fatal msg="failed to fetch Cluster: failed to generate asset \"Cluster\": failed to create cluster: failed to apply Terraform: failed to complete the change"
    level=fatal msg="failed to get bootstrap and control plane host addresses from \"terraform.tfstate\": no bootstrap instance found"
```